app/services/document.ts

```diff
@@ -100,12 +101,15 @@ export default class DocumentService extends Service {
         this.loadingStatus = `${message}`;
       };
 
-      return await validatePublication(
+      const result = await validatePublication(
         this.document,
         blueprint,
         example,
         onProgress,
       );
+      console.log(result);
```

Comment:
Suggestion: remove console log?

Reply:
I'm leaving it, as this is valuable for the pro user of the validation tool